lizard_downloader.py

```diff
@@ -216,7 +217,7 @@ def show_data(self):
         """ Show the data as a new layer on the map """
 
         # Setup
-        WGS84 = "EPSG:4326"
+        wgs84 = "EPSG:4326"
```

Comment:
I changed it into lower case, because that would be more in accordance with PEP8.
But if upper case is the convention for constants, I'll change it.